content/hi/cloud_native_security.md

```diff
@@ -0,0 +1,25 @@
+--- 
+
+Title: क्लाउड नेटिव सुरक्षा (Cloud Native Security)
+Status: completed 
+Category: अवधारणा 
+
+--- 
+
+ 
+
+## यह क्या है 
+
+क्लाउड नेटिव सुरक्षा एक ऐसा दृष्टिकोण है जो सुरक्षा को [क्लाउड नेटिव एप्लिकेशन](/cloud_native_apps/) में बनाता है। यह सुनिश्चित करता है कि सुरक्षा विकास के संपूर्ण एप्लिकेशन जीवनचक्र का हिस्सा है। क्लाउड नेटिव सुरक्षा पारंपरिक सुरक्षा मॉडल के समान मानकों को सुनिश्चित करने का प्रयास करती है, जबकि क्लाउड नेटिव वातावरण के विवरणों को अपनाती है, अर्थात् तेजी से कोड परिवर्तन और अत्यधिक अल्पकालिक बुनियादी ढाँचा। क्लाउड नेटिव सुरक्षा `DevSecOps` नामक अभ्यास से अत्यधिक संबंधित है। 
+
+ 
+
+## समस्या 
+
+पारंपरिक सुरक्षा मॉडल कई मान्यताओं के साथ बनाए गए थे जो अब मान्य नहीं हैं। क्लाउड नेटिव ऐप बार-बार बदलते हैं, बड़ी संख्या में ओपन सोर्स टूल और लाइब्रेरी का उपयोग करते हैं, जो अक्सर विक्रेता-नियंत्रित बुनियादी ढांचे में चलते हैं, और तेजी से बुनियादी ढांचे में बदलाव के अधीन हैं। कोड समीक्षाएं, लंबे गुणवत्ता आश्वासन चक्र, होस्ट-आधारित भेद्यता स्कैनिंग, और अंतिम मिनट की सुरक्षा समीक्षाएं क्लाउड नेटिव एप्लिकेशन के साथ स्केल नहीं हो सकती हैं। 
+
+ 
+
+## समाधान 
+
+क्लाउड नेटिव सुरक्षा काम करने का एक नया तरीका पेश करती है जो पारंपरिक सुरक्षा मॉडल से एक में माइग्रेट करके अनुप्रयोगों की सुरक्षा करती है जहां सुरक्षा रिलीज चक्र के हर चरण में शामिल होती है। मैनुअल ऑडिट और चेक को बड़े पैमाने पर स्वचालित स्कैन से बदल दिया जाता है। रैपिड कोड रिलीज़ पाइपलाइनों को ऐसे टूल के साथ एकीकृत किया जाता है जो संकलित होने से पहले कमजोरियों के लिए कोड स्कैन करते हैं। ओपन सोर्स लाइब्रेरी को विश्वसनीय स्रोतों से खींचा जाता है और कमजोरियों के लिए निगरानी की जाती है। परिवर्तन को धीमा करने के बजाय एक क्लाउड नेटिव सुरक्षा मॉडल इसे बार-बार अपडेट किए जाने वाले कमजोर घटकों द्वारा शामिल करता है या यह सुनिश्चित करता है कि बुनियादी ढांचे को नियमित रूप से बदल दिया जाए। 

```

Comment:
The first sentence seems a little misplaced. Here's a suggestion to write it in a different way :
```क्लाउड नेटिव सुरक्षा पारंपरिक सुरक्षा मॉडल को एक ऐसे मॉडल में माइग्रेट करती है जहां सुरक्षा रिलीज़ चक्र के हर चरण में शामिल होती है, इस प्रकार एप्लिकेशन की सुरक्षा का एक नया तरीका पेश करती है।```

Reply:
Fixed, let me know any changes!!

```suggestion
क्लाउड नेटिव सुरक्षा पारंपरिक सुरक्षा मॉडल को एक ऐसे मॉडल में माइग्रेट करती है जहां सुरक्षा रिलीज़ चक्र के हर चरण में शामिल होती है, इस प्रकार एप्लिकेशन की सुरक्षा का एक नया तरीका पेश करती है। मैनुअल ऑडिट और चेक को बड़े पैमाने पर स्वचालित स्कैन से बदल दिया जाता है। रैपिड कोड रिलीज़ पाइपलाइनों को ऐसे टूल के साथ एकीकृत किया जाता है जो संकलित होने से पहले कमजोरियों के लिए कोड स्कैन करते हैं। ओपन सोर्स लाइब्रेरी को विश्वसनीय स्रोतों से खींचा जाता है और कमजोरियों के लिए निगरानी की जाती है। परिवर्तन को धीमा करने के बजाय एक क्लाउड नेटिव सुरक्षा मॉडल इसे बार-बार अपडेट किए जाने वाले कमजोर घटकों द्वारा शामिल करता है या यह सुनिश्चित करता है कि बुनियादी ढांचे को नियमित रूप से बदल दिया जाए। 

```

Reply:
looks good